website/content/sdk/installation.md

```diff
@@ -46,9 +48,19 @@ and
 pip install openbb[forecast]
 ```
 
+Install all available toolkits at once with:
+
+```shell
+pip install openbb[all]
+```
+
+:::info
+If you encounter any issues with the installation, please try following the instructions to install OpenBB Terminal from source [(link)](/terminal/installation/source).
+:::
+
 ## Verify Installation
 
-Confirm the installation by opening the python interpreter with a `python` command and running the following:
+To confirm the installation, activate your virtual environment and open the Python interpreter with a python command, then run the following:
```

Comment:
This is also pretty unclear to me. In essence this is just open a command line tool, activate the environment and type `python`. Those steps should be displayed. Then proceed with the things below. Also in Windows, you'd need the "Anaconda Prompt" if you don't set any envs, that would need to be clear (or ruled out entirely that that is a possibility).

Reply:
same comment as above We don't know what virtual environment technology the person is using. It can be venv pipenv conda etc. We advise following the terminal source docs - they explain how to activate a conda environment